fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java

```diff
@@ -2384,5 +2385,18 @@ private void initTableFunction() {
                 "_ZN5doris19DummyTableFunctions22explode_numbersEPN9doris_udf15FunctionContextERKNS1_9IntValE",
                 null, null, true));
         tableFunctions.put(EXPLODE_NUMBERS, explodeNumbers);
+
+        List<Function> explodes = Lists.newArrayList();
+        explodes.add(ScalarFunction.createBuiltin(
+                EXPLODE_OUTER, Type.INT, Function.NullableMode.DEPEND_ON_ARGUMENT,
```

Comment:
The 'explode_outer' function should be 'always_nullable'

Reply:
done